aya/src/bpf.rs

```diff
@@ -229,7 +262,7 @@ impl<'a> BpfLoader<'a> {
                 }
                 PinningType::None => map.create(&name)?,
             };
-            if !map.obj.data.is_empty() && name != ".bss" {
+            if !map.obj.data.is_empty() && map.obj.kind != MapKind::Bss {
```

Comment:
I think the bss check needs to stay? - bss shouldn't be initialized (it's the whole point)

Reply:
The check is still there. We're just using an enum variant now instead.

Reply:
I'm sorry I think I reviewed some stale files, I don't know how i managed to get vscode in that state